test/functional/test_runner.py

```diff
@@ -157,6 +157,21 @@
 # Place EXTENDED_SCRIPTS first since it has the 3 longest running tests
 ALL_SCRIPTS = EXTENDED_SCRIPTS + BASE_SCRIPTS
 
+def check_script_prefixes():
```

Comment:
Consider adding a doc string for this function:

```py
"""Check that no more than `EXPECTED_VIOLATION_COUNT` of the test scripts don't start with one of the allowed name prefixes."""
```

Reply:
doc string added